docs/layers/tile-layer.md

```diff
@@ -173,19 +174,37 @@ How the tile layer refines the visibility of tiles. One of the following:
 
 - Default: `'best-available'`
 
+##### `throttleRequests` (Boolean, optional)
+
+Whether to throttle `getTileData` requests.
+
+If `true`, a maximum of `maxRequests` instances of `getTileData` will be called concurrently. Additionally, for requests that are throttled, if the tile is no longer visible in the viewport when a request slot opens up, that request will be cancelled. When requests are not throttled, `getTileData` will be called for all visible tiles, regardless of how long that tile is or was visible.
+
+- Default: `true`
+
+##### `maxRequests` (Number, optional)
+
+The maximum number of concurrent `getTileData` calls. Only takes effect if `throttleRequests` is `true`. 
+
+- Default: `8`
 
 ### Render Options
 
-##### `renderSubLayers` (Function, optional))
+##### `renderSubLayers` (Function, optional)
 
 Renders one or an array of Layer instances with all the `TileLayer` props and the following props:
 
 * `id`: An unique id for this sublayer
-* `data`: Resolved from `getTileData`
+* `data`: Resolved from `getTileData`. As of deck.gl 8.2, this prop is always the data resolved from the Promise and is never a Promise itself.
 * `tile`: An object containing tile index `x`, `y`, `z`, and `bbox` of the tile.
 
 - Default: `props => new GeoJsonLayer(props)`
 
+##### `zRange` (Array, optional)
+
+An array representing the range of minimum and maximum heights in the tile. This is designed to support extruded layers, such as the TerrainLayer, ensuring the right tiles are loaded and rendered. This prop currently only has effect when used with a geospatial view.
```

Comment:
I believe it is respected in non-geospatial as well.

Reply:
From [here](https://github.com/visgl/deck.gl/blob/d96e91f56f2c120d59c1c5ee2832848bed9be139/modules/geo-layers/src/tile-layer/utils.js#L140-L142) I assumed it was only geospatial